src/main/java/spireCafe/patches/CafeEntryExitPatch.java

```diff
@@ -101,6 +101,8 @@ public static class CafeExitMarkCompletePatch {
         public static void markComplete(ProceedButton proceedButton) {
             if (inCafe()) {
                 AbstractDungeon.currMapNode.room.phase = AbstractRoom.RoomPhase.COMPLETE;
+                AbstractDungeon.player.drawX = (float)Settings.WIDTH * 0.25F;
```

Comment:
Please save and reset the position. I don't think it's a good idea to just wing it